websites/P/Pandora/dist/metadata.json

```diff
@@ -21,7 +21,7 @@
     "nl": "Speel de nummers, albums, afspeellijsten en podcasts af die je leuk vindt op de geheel nieuwe Pandora."
   },
   "url": "www.pandora.com",
-  "version": "2.0.2",
+  "version": "2.0.3",
```

Comment:
```suggestion
  "version": "2.1.0",
```

Reply:
I was thinking about this and was like, yeah this seems like a minor less bugfix change haha